### PR TITLE
fix last releases todo and lints

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
- fix last releases todo and lints

Fixes: https://github.com/sigstore/timestamp-authority/issues/50

rehearsal: https://github.com/cpanato/timestamp-authority/releases/tag/v99.99.02

```
$ COSIGN_EXPERIMENTAL=1 cosign verify ghcr.io/cpanato/timestamp-server:v99.99.02

Verification for ghcr.io/cpanato/timestamp-server:v99.99.02 --
The following checks were performed on each of these signatures:
  - The cosign claims were validated
  - Existence of the claims in the transparency log was verified offline
  - Any certificates were verified against the Fulcio roots.

[{"critical":{"identity":{"docker-reference":"ghcr.io/cpanato/timestamp-server"},"image":{"docker-manifest-digest":"sha256:376c6375a9dd20275bcee2bc11ef25ce1220b6546655b9ce8347edb7b85b48eb"},"type":"cosign container image signature"},"optional":{"1.3.6.1.4.1.57264.1.1":"https://token.actions.githubusercontent.com","1.3.6.1.4.1.57264.1.2":"push","1.3.6.1.4.1.57264.1.3":"645dec9bd52385d6712c7cf998c72ffdc38eb717","1.3.6.1.4.1.57264.1.4":"cut-release","1.3.6.1.4.1.57264.1.5":"cpanato/timestamp-authority","1.3.6.1.4.1.57264.1.6":"refs/tags/v99.99.02","Bundle":{"SignedEntryTimestamp":"MEUCID9n/Ztc2yxXTFLJrI34PrIhye8+a03gW84NPgDEUvlDAiEAg+KjloauFN89NNwmRtPsomIbPEyQ6H5t/ClcRpjwW5I=","Payload":{"body":"eyJhcGlWZXJzaW9uIjoiMC4wLjEiLCJraW5kIjoiaGFzaGVkcmVrb3JkIiwic3BlYyI6eyJkYXRhIjp7Imhhc2giOnsiYWxnb3JpdGhtIjoic2hhMjU2IiwidmFsdWUiOiI1YzJiODEzZjUwMzBjOGFjNTFjNGI5NzBmNjNkNjU4N2U0ZjdhNzljMDg4MjExMzg4NzFkZjQxYmU0MjQ5YmZmIn19LCJzaWduYXR1cmUiOnsiY29udGVudCI6Ik1FVUNJSFlpSWoxemsvOHdhYXRhQmx6ZnB0WDNOaWtCRW1BamlVWWVvNFIwN2hiK0FpRUFtR3V4NGhUSkRJZVRWQVFyRlF2S1F6aDQyanhHRkUrd1NGQ3Ira0VqWGFzPSIsInB1YmxpY0tleSI6eyJjb250ZW50IjoiTFMwdExTMUNSVWRKVGlCRFJWSlVTVVpKUTBGVVJTMHRMUzB0Q2sxSlNVUnpSRU5EUVhwcFowRjNTVUpCWjBsVlRsQnZRMWRET0ZocWFXTTNaMmszZW5wMFJGcHZkWFpKY0hsamQwTm5XVWxMYjFwSmVtb3dSVUYzVFhjS1RucEZWazFDVFVkQk1WVkZRMmhOVFdNeWJHNWpNMUoyWTIxVmRWcEhWakpOVWpSM1NFRlpSRlpSVVVSRmVGWjZZVmRrZW1SSE9YbGFVekZ3WW01U2JBcGpiVEZzV2tkc2FHUkhWWGRJYUdOT1RXcEplRTFFUlhsTlZHdDZUbnBOTTFkb1kwNU5ha2w0VFVSRmVVMVVhekJPZWswelYycEJRVTFHYTNkRmQxbElDa3R2V2tsNmFqQkRRVkZaU1V0dldrbDZhakJFUVZGalJGRm5RVVZpVTFwaVUzcE1ORXA1VW5OYU4wUkpRbHBzUm5sTFppOXpWVTl0V1dFcldVdFhXa0lLWTJodFQzaFlReXN3ZFZsVWMxQk1aWEV2Um5KaVFsSXlXRGR4YVVsV1ltRm5SRVV2ZGtoNlp6QndXRkZxV1RJeVdYRlBRMEZzWTNkblowcFVUVUUwUndwQk1WVmtSSGRGUWk5M1VVVkJkMGxJWjBSQlZFSm5UbFpJVTFWRlJFUkJTMEpuWjNKQ1owVkdRbEZqUkVGNlFXUkNaMDVXU0ZFMFJVWm5VVlZNWWtReUNuaGFabEl5YUhaSmNFMWpjQ3RCT1dneU5qaFplWE5yZDBoM1dVUldVakJxUWtKbmQwWnZRVlV6T1ZCd2VqRlphMFZhWWpWeFRtcHdTMFpYYVhocE5Ga0tXa1E0ZDJKM1dVUldVakJTUVZGSUwwSkhWWGRaTkZwb1lVaFNNR05JVFRaTWVUbHVZVmhTYjJSWFNYVlpNamwwVERKT2QxbFhOV2hrUnpoMlpFZHNkQXBhV0U0d1dWY3hkMHhYUmpGa1IyaDJZMjFzTUdWVE9IVmFNbXd3WVVoV2FVd3paSFpqYlhSdFlrYzVNMk41T1hsYVYzaHNXVmhPYkV4dWJHaGlWM2hCQ21OdFZtMWplVGt3V1Zka2Vrd3pXVFZQVXpRMVQxTTBkMDFxUVRWQ1oyOXlRbWRGUlVGWlR5OU5RVVZDUWtOMGIyUklVbmRqZW05MlRETlNkbUV5Vm5VS1RHMUdhbVJIYkhaaWJrMTFXakpzTUdGSVZtbGtXRTVzWTIxT2RtSnVVbXhpYmxGMVdUSTVkRTFDU1VkRGFYTkhRVkZSUW1jM09IZEJVVWxGUWtoQ01RcGpNbWQzVG1kWlMwdDNXVUpDUVVkRWRucEJRa0YzVVc5T2FsRXhXa2RXYWs5WFNtdE9WRWw2VDBSV2EwNXFZM2hOYlUweldUSlpOVTlVYUdwT2VrcHRDbHB0VW1wTmVtaHNXV3BqZUU1NlFWcENaMjl5UW1kRlJVRlpUeTlOUVVWRlFrRjBhbVJZVVhSamJWWnpXbGRHZWxwVVFYQkNaMjl5UW1kRlJVRlpUeThLVFVGRlJrSkNkR3BqUjBaMVdWaFNka3d6VW5CaVYxWjZaRWRHZEdORE1XaGtXRkp2WWpOS2NHUklhM2RKVVZsTFMzZFpRa0pCUjBSMmVrRkNRbWRSVkFwamJWWnRZM2s1TUZsWFpIcE1NMWsxVDFNME5VOVROSGROYWtOQ2FXZFpTMHQzV1VKQ1FVaFhaVkZKUlVGblVqaENTRzlCWlVGQ01rRkJhR2RyZGtGdkNsVjJPVzlTWkVoU1lYbGxSVzVGVm01SFMzZFhVR05OTkRCdE0yMTJRMGxIVG0wNWVVRkJRVUpuT0RKNk1HWjNRVUZCVVVSQlJXTjNVbEZKWjBoTVpGWUtRa1JGVW1WU2VtUm5hRGhTY0ZCVlNtdG9UVFJwU1RGVVNUWk1kVkl6VEVWcGJWWndOM3BqUTBsUlEydGhZVUV6WVc4clRUSjFhRFl4YUhSeloyUlFkd3BVVFhORGJGbDRVQzlEY1VsTE1rSjBaelZPUzI1RVFVdENaMmR4YUd0cVQxQlJVVVJCZDA1dFFVUkNha0ZxUWtOVVExRXlNbk50YlVNd1IyOU5NRFpzQ21RME5tbFVWVEJwY0dkbU9EVnBRVlJtYURKRk5XNU5OVzUwVVhOVVpVWkRkV052Wld4U1kwNUJNMFJ3WlN0clEweDNZa0U0Vm14WWVGWnRlVWgyY0hvS01VOUxiWE5GV0dJcmNYbFlRMjFYVlc1TllsQndTVU5xYlhBeU0wMUVSM1JYTURCdE9HMTNMMjV5UkRGUlVtRjFDaTB0TFMwdFJVNUVJRU5GVWxSSlJrbERRVlJGTFMwdExTMEsifX19fQ==","integratedTime":1665603459,"logIndex":4981596,"logID":"c0d23d6ad406973f9559f3ba2d1ca01f84147d8ffc5b8445c224f98b9591801d"}},"GIT_HASH":"645dec9bd52385d6712c7cf998c72ffdc38eb717","GIT_VERSION":"v99.99.02","Issuer":"https://token.actions.githubusercontent.com","Subject":"https://github.com/cpanato/timestamp-authority/.github/workflows/release.yaml@refs/tags/v99.99.02","githubWorkflowName":"cut-release","githubWorkflowRef":"refs/tags/v99.99.02","githubWorkflowRepository":"cpanato/timestamp-authority","githubWorkflowSha":"645dec9bd52385d6712c7cf998c72ffdc38eb717","githubWorkflowTrigger":"push"}}]
```

### CODE REVIEW

File: release/ko-sign-release-images.sh [hunk 1]
```diff
@@ -32,5 +32,7 @@ if [[ ! -f timestampCLIImagerefs ]]; then
 fi
 
 echo "Signing images with Keyless..."
-cosign sign --force -a GIT_HASH="$GIT_HASH" -a GIT_VERSION="$GIT_VERSION" "$(cat timestampServerImagerefs)"
-cosign sign --force -a GIT_HASH="$GIT_HASH" -a GIT_VERSION="$GIT_VERSION" "$(cat timestampCLIImagerefs)"
+readarray -t server_images < <(cat timestampServerImagerefs || true)
+cosign sign --force -a GIT_HASH="${GIT_HASH}" -a GIT_VERSION="${GIT_VERSION}" "${server_images[@]}"
+readarray -t cli_images < <(cat timestampCLIImagerefs || true)
+cosign sign --force -a GIT_HASH="${GIT_HASH}" -a GIT_VERSION="${GIT_VERSION}" "${cli_images[@]}"
```
Comment (haydentherapper) [inline release/ko-sign-release-images.sh:38]: Would it be useful to add a verification step too just to double check the signing worked?

Comment (cpanato) [reply release/ko-sign-release-images.sh:38]: like `cosign verify` ?

Comment (cpanato) [reply release/ko-sign-release-images.sh:38]: ok

Comment (haydentherapper) [reply release/ko-sign-release-images.sh:38]: Yep, `cosign verify`, just as a double check

File: .ko.yaml [hunk 2]
```diff
@@ -14,35 +14,35 @@
 # limitations under the License.
 
 # We need a shell for a lot of redirection/piping to work
-defaultBaseImage: ghcr.io/distroless/static:latest
```
Comment (haydentherapper) [inline .ko.yaml:17]: I recognize that this has moved, but to be honest, I'd prefer that that name of the base image be vendor neutral. Are there any distroless base images that are vendor neutral? I'm not very knowledgable in this space, so whatever you think is best.

**Comment by haydentherapper** [comment]
Do you want to copy in the changes from https://github.com/sigstore/fulcio/pull/815 into here too?

File: .ko.yaml (hunk 2 above)
Comment (cpanato) [reply .ko.yaml:17]: we were using that before, it just got renamed, but if you dont want we can swtich to the google distroless one, but imho i think this is more up to date :)

**Comment by cpanato** [comment]
> Do you want to copy in the changes from [sigstore/fulcio#815](https://github.com/sigstore/fulcio/pull/815) into here too?

will do in a follow up :)

File: .ko.yaml (hunk 2 above)
Comment (haydentherapper) [reply .ko.yaml:17]: I'm fine to use this! Just wanted to mention this since it might come up again.

**Comment by cpanato** [comment]
@haydentherapper done and new rehearsal are here https://github.com/cpanato/timestamp-authority/releases/tag/v99.99.03 with the verify signatures of the images in place

**Review by haydentherapper — APPROVED** [review]
Fantastic!! Thank you so much!

**Pull request merged**